waspc/examples/todoApp/src/queries.ts

```diff
@@ -47,3 +58,11 @@ export const getTask: GetTask<Pick<Task, 'id'>, Task> = async (where, context) =
 export const getDate: GetDate<void, Date> = async () => {
   return new Date()
 }
+
+export const getAnything: GetAnything = async () => {
+  return 'anything'
+}
+
+export const getTrueVoid = (async () => {
+  return 'anything'
+}) satisfies GetTrueVoid
```

Comment:
This is all that's new. I've added these Queries to cover all edge cases with my types tests in `TestRpcTypes`.

The rest of the file is formatting.